example/sigma/derive_zmu.jl

```diff
@@ -28,16 +30,25 @@ if abspath(PROGRAM_FILE) == @__FILE__
         isSave = true
     end
 
+<<<<<<< HEAD
```

Comment:
Please clean up this merge by replacing lines 33-39 with `f = jldopen(sigma_z_filename, "r")`